content/en/contribute/code/workflow.md

```diff
@@ -10,7 +10,7 @@ description: >
 
 ### Writing
 
-Where possible, follow our coding [style guide]({{< ref "docs-style-guide" >}}).
+Where possible, follow our coding [style guide]({{< ref "contribute/docs/style-guide" >}}).
```

Comment:
As @mrjones-plip noticed in #165 , this should link to the coding style guide not the docs style guide.

```suggestion
Where possible, follow our [coding style guide]({{< ref "contribute/code/style-guide" >}}).
```